### PR TITLE
Personal info details screen

### PR DESCRIPTION
**Pull request by Oleh-Li**
please check

### CODE REVIEW

File: src/components/personalInfoPage/MainPersonalInfo.js [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+        <View style={styles.titleAndInputWrapper}>
+          <Text style={styles.titleInfo}>Mobile Phone</Text>
+          <MobileIconSvg style={styles.inputIcon} />
+          <TextInput
+            style={styles.textInputBorderFocus}
+            placeholder={"Enter your mobile number"}
+          ></TextInput>
+        </View>
+
+        <View style={[styles.titleAndInputWrapper, { marginBottom: 27 }]}>
```
Comment (osavchukdoit) [inline src/components/personalInfoPage/MainPersonalInfo.js:95]: Lets extract these inline styles to some named style properties.

**Review by osavchukdoit — CHANGES_REQUESTED** [review]
Pay attention to the comments.

File: src/screens/PersonalInfoDetailsScreen.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <ScrollView stickyHeaderIndices={[0]}>
+        <TopPersonalInfo />
+        <MainPersonalInfo />
+      </ScrollView>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  wrapper: {
```
[the remark this answers is not in the document]
Comment (Oleh-Li) [reply src/screens/PersonalInfoDetailsScreen.js]: yes, it was redundant

**Comment by Oleh-Li** [comment]
fixed all in last commit, please check

**Review by osavchukdoit — APPROVED** [review]
Good.

**Pull request merged**